composer.json

```diff
@@ -21,7 +21,8 @@
         "mockery/mockery": "^0.9.4|^1.3.1",
         "mtdowling/burgomaster": "dev-master#72151eddf5f0cf101502b94bf5031f9c53501a04",
         "phpunit/phpunit": "^4.8.36|^7.5.15",
-        "php-mock/php-mock-phpunit": "^1.1|^2.1"
+        "php-mock/php-mock-phpunit": "^1.1|^2.1",
+        "sebastian/version": ">=1.0.3"
```

Comment:
Missing upper bound.

Reply:
I'm not sure if we need an upper bound here — we don't use this directly but go via PHPUnit's `PHPUnit\Runner\Version` class so BC breaks shouldn't affect us unless they're also BC breaks in PHPUnit, which we'd need to fix anyway. It also doesn't really change; the only new release in the last 4 years was essentially a PHP version bump

We just need a lower bound when using `--prefer-lowest` to avoid [a bug in v1.0.0-1.0.2](https://github.com/sebastianbergmann/version/issues/7) where it reads our version number rather than PHPUnit's

Reply:
`^1.0.3` then?